stanford_events.libraries.yml

```diff
@@ -46,6 +46,10 @@ event_node:
   css:
     component:
       dist/css/stanford_events.node.css: {}
+      dist/css/stanford_events.event-list.css: {}
+      dist/css/stanford_events.event-card.css: {}
```

Comment:
These should not be included here. @jenbreese the idea is the list and the card styles are added to those patterns/views. Basically we only want to add those styles if the list and cards are present.

Reply:
Ok, I'm confused. :)  The card are on the node page. The event list css has all the styles for the dates. This would keep styles from being duplicated to different css files, right?

Reply:
An option would be to have a scss partial with those date styles and import those into the different style sheets. the idea is that when on an event node page, if there are no event cards at the bottom, it doesn't load the event cards and list styles that are not necessary.

Reply:
Ok. Here's my next pass at it. :)